### PR TITLE
samples: radio_test: Align with pinctrl

### PR DESCRIPTION
**Pull request by KAGA164**
Align the Radio Test sample configuration with
pinctrl.

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -165,7 +165,8 @@ Zigbee samples
 Other samples
 -------------
 
-|no_changes_yet_note|
+* :ref:`radio_test` sample:
+   * Fixed setting gain for nRF21540 Front-end Module with nRF5340.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:169]:
```suggestion
   * Fixed the way of setting gain for the nRF21540 Front-end Module with nRF5340.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

**Review by peknis — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -327,7 +327,12 @@ nRF5
 nRF5340
 =======
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-0-0
```
Comment (grochu) [inline doc/nrf/known_issues.rst:330]: Probably 2.0.1 and 2.0.2 as derived directly from 2.0.0 are also affected?
```suggestion
.. rst-class:: v2-0-2 v2-0-1 v2-0-0
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -395,7 +395,8 @@ Zigbee samples
 Other samples
 -------------
 
-|no_changes_yet_note|
+* :ref:`radio_test` sample:
+   * Fixed the way of setting gain for the nRF21540 Front-end Module with nRF5340.
```
Comment (grochu) [inline doc/nrf/releases/release-notes-changelog.rst:399]: How about if we simplify a bit:
```suggestion
   * Fixed the gain setting for the nRF21540 Front-end Module with nRF5340.
```

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -327,7 +327,12 @@ nRF5
...
+
+NCSDK-16338: Setting gain for nRF21540 Front-end module does not work in the :ref:`radio_test` sample.
```
Comment (grochu) [inline doc/nrf/known_issues.rst:332]:
```suggestion
NCSDK-16338: Setting gain for nRF21540 Front-end module does not work in the :ref:`radio_test` sample on nRF5340.
```

**Review by grochu — DISMISSED** [review]

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (KAGA164) [reply doc/nrf/known_issues.rst]: Yes they are affected also. Added

**Review by kapi-no — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]

**Pull request merged**